modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -3624,18 +3624,17 @@ export class Wallet implements IWallet {
   }
 
   /**
-   * Send funds from a fee address to a forwarder. Only supports eth-like coins.
+   * Send funds from a fee address to a forwarder.
    *
    * @param {Object} params - parameters object
-   * @param {String} params.forwarderAddress - Address of the forwarder to send funds to.
-   * @param {String} params.amount - Amount to send the forwarder (optional). If not given, defaults to sending an estimate of the amount needed for a fund recovery
+   * @param {List} params.forwarders - list of fund forwarder options
    * @returns {*}
    */
-  public async fundForwarder(params: FundForwardersOptions): Promise<any> {
-    if (_.isUndefined(params.forwarderAddress)) {
-      throw new Error('forwarder address required');
+  public async fundForwarders(params: FundForwarderParams): Promise<any> {
```

Comment:
This could be a breaking change and could break existing client integration, Was the change made in API?, I would suggest if its a recent change in API, fix the breaking change from API

Reply:
This API doesn't exist in express server. Most probably this implementation is not being used. May be I'll not modify it and add a new method instead.

Reply:
Added a new method instead of modifying the existing one.

Reply:
```suggestion
  public async fundForwarders(params: FundForwarderParams): Promise<WalletData> {
```

Reply:
we can type this one